Makefile

```diff
@@ -1,22 +1,42 @@
 # Current Operator version
 VERSION ?= 0.0.1
 
+# Current bundle version
+BUNDLE_VERSION ?= 1.0.0
+
 .PHONY: \
 	clean \
 	deploy \
 	func-test \
 	unit-test \
-	build
+	build \
+	manifests \
+	gen-for-bundle \
+	bundle \
+	bundle-build
 
 SOURCES := $(shell find . -name '*.go' -not -path "*/vendor/*")
 GOBUILDFLAGS ?= -i -mod=vendor
 GOLDFLAGS ?= -s -w -X github.com/openshift/cincinnati-operator/version.Operator=$(VERSION)
 
 # This is a placeholder for cincinnati-operator image placeholder
 # During development override this when you want to use an specific image
-# Example: IMG ?= quay.io/jottofar/updateservice-operator-index:v1
+# Example: IMG ?= quay.io/jottofar/update-service-operator:v1
 IMG ?= controller:latest
 
+BUNDLE_IMG ?= controller-bundle:latest
+
+CHANNELS ?= v1
+BUNDLE_CHANNELS = --channels=$(CHANNELS)
```

Comment:
SDK scaffolding [has some `ifdefs` around `BUNDLE_CHANNELS` and `BUNDLE_DEFAULT_CHANNEL`][1].  Do we know what those are about?  They also don't seem to have a default for `CHANNELS`.  Do we understand the flow for that?  Is it something that's usually set explicitly at build time?

[1]: https://github.com/openshift/cincinnati-operator/blob/0731f08b2fbd3989b4642ff55c6a216e97d4c990/Makefile#L6-L12

Reply:
Channels are very much like our release candidate, fast, stable channels -> "Channels allow package authors to write different upgrade paths for different users (e.g. alpha vs. stable)"

I was trying to keep it simple for the first release and figured we would only have the one release initially.